assets/xml/objects/object_cne.xml

```diff
@@ -16,7 +16,7 @@
         <!-- <Limb Name="object_cne_Standardlimb_000090" Type="Standard" Offset="0x90" /> -->
         <!-- <Limb Name="object_cne_Standardlimb_00009C" Type="Standard" Offset="0x9C" /> -->
         <!-- <Limb Name="object_cne_Standardlimb_0000A8" Type="Standard" Offset="0xA8" /> -->
-        <!-- <Skeleton Name="object_cne_Skel_0000F0" Type="Flex" LimbType="Standard" Offset="0xF0" /> -->
+        <!-- <Skeleton Name="object_cne_Skel_0000F0" Type="Flex" LimbType="Standard" LimbNone="_LIMB_NONE" LimbMax="_LIMB_MAX" EnumName="_Limbs" Offset="0xF0" /> -->
```

Comment:
Is this intended? I'm not sure what ZAPD would do with this assuming this was uncommented. I suppose it doesn't *really* matter since, whenever the unaligned textures issue is solved, someone is going to have to submit a PR to uncomment things in this file, and in doing so they can fix this enum.

Reply:
No, it isn't.
I guess I can fix this one manually